### PR TITLE
fix Part of  #2833 :Add missing named Argument in ProfileListFragmentTest

### PR DESCRIPTION
**Pull request by Naman-1234**
## Explanation

Fixes Part of #2833 :Add missing named Argument in ProfileListFragmentTest

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by anandwana001** [comment]
@Naman-1234 If you are not using Windows, please complete all the installation process - https://github.com/oppia/oppia-android/wiki#installation

**Comment by Naman-1234** [comment]
@anandwana001,Sir I am using windows.  Could you please tell me how should I proceed?

**Comment by anandwana001** [comment]
You have to fix the formatting errors

```
********************************
Checking code formatting
********************************
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:122:24: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:122:57: Trailing space(s)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:123:19: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:123:22: Trailing space(s)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:124:25: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:124:47: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:130:24: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:131:19: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:131:22: Trailing space(s)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:132:23: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:132:51: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:143:24: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:144:19: Missing spacing before "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:145:25: Missing spacing before "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:145:48: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:151:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:151:22: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:152:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:152:17: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:153:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:153:21: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:153:49: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:173:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:173:22: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:174:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:174:17: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:175:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:175:23: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:175:45: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:181:1: Unexpected indentation (9) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:181:23: Missing spacing before "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:182:1: Unexpected indentation (9) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:182:18: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:182:21: Trailing space(s)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:183:1: Unexpected indentation (9) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:183:22: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:183:50: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:194:1: Unexpected indentation (9) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:194:23: Missing spacing before "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:195:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:195:17: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:196:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:196:23: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:196:45: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:202:1: Unexpected indentation (9) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:202:23: Missing spacing before "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:203:1: Unexpected indentation (9) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:203:18: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:203:21: Trailing space(s)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:204:1: Unexpected indentation (9) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:204:22: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:204:50: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:224:24: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:225:19: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:226:25: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:226:47: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:237:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:237:22: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:238:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:238:17: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:239:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:239:23: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:239:45: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:250:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:250:22: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:251:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:251:17: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:252:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:252:23: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:252:45: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:263:24: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:264:19: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:265:25: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:265:47: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:276:24: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:277:19: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:278:25: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:278:47: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:289:1: Unexpected indentation (9) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:289:23: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:290:1: Unexpected indentation (9) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:290:18: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:291:1: Unexpected indentation (8) (should be 10)
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:291:23: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:291:45: Missing newline before ")"
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:302:24: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:303:19: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:304:25: Missing spacing around "="
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:304:47: Missing newline before ")"
********************************
Ktlint issue found.
Please fix the above issues.
  You can also use the ktlint -F --android domain/src/**/*.kt utility/src/**/*.kt data/src/**/*.kt app/src/**/*.kt testing/src/**/*.kt
  command to fix the most common issues.
Please note, there might be a possibility where the above command will not fix the issue.
  In that case, you will have to fix it yourself.
********************************
```

Reference - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide (Use Macros)

As these errors are just the formatting so, I think you can skip installing ktlint on your system and simply fix using the above error as it is mentioned.

**Comment by Naman-1234** [comment]
Sure, Sir, I Will fix this in the latest commit.

**Comment by Naman-1234** [comment]
@anandwana001, Sir I am so sorry to disturb you but I am getting this error while running bash scripts/setup.sh and also during pushing of the code. I tried to search it in the error resolving part given in the wiki installation portion. But didn't find anything. Could you please help me with this. Please find the error of the screenshot here:


Screenshot (1088)

**Comment by anandwana001** [comment]
Yeah, we have an ongoing PR for this issue. 
What you can do right now is to go to the `.git` folder inside the oppia-android, it's a hidden folder. Inside it, you can find a holder named `hooks` and remove the file called `pre-push.sh`. And then you can simply push it. 
Let me know if you need any assistance in this.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Please check line number 222, passing argument 5 doesn't make it readable, so we can have a name argument to make it more readable.

**Review by anandwana001 — CHANGES_REQUESTED** [review]
The argument name should be the same as the name of the recieveing argument.

Reference - https://kotlinlang.org/docs/functions.html#named-arguments

**Comment by Naman-1234** [comment]
@anandwana001, Working on changes sir.

**Comment by Naman-1234** [comment]
@anandwana001, Sir Please review this PR and please let me know if there are any changes required.

File: app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt [hunk 1]
```diff
@@ -187,7 +219,7 @@ class ProfileListFragmentTest {
   @Test
   fun testProfileListFragment_addManyProfiles_checkProfilesAreSorted() {
     profileTestHelper.initializeProfiles()
-    profileTestHelper.addMoreProfiles(5)
+    profileTestHelper.addMoreProfiles(numberOfProfiles=5)
```
Comment (FareesHussain) [inline app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt]:
```suggestion
    profileTestHelper.addMoreProfiles(numberOfProfiles = 5)
```

```
/home/runner/work/oppia-android/oppia-android/app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:222:55: Missing spacing around "="
```

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@Naman-1234 NIT fix related to lint check, rest looks good

File: app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt (hunk 1 above)
Comment (Naman-1234) [reply app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt]: Changes done.

Comment (Naman-1234) [reply app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt]: Please let me know if changes are Ok now or do any change required?

File: .project [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			<id>1615142726842</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>
```
Comment (FareesHussain) [inline .project]: Please revert these changes

File: .settings/org.eclipse.buildship.core.prefs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
+connection.project.dir=
+eclipse.preferences.version=1
+gradle.user.home=
+java.home=C\:/Program Files/Amazon Corretto/jdk11.0.8_10
+jvm.arguments=
+offline.mode=false
+override.workspace.settings=true
+show.console.view=true
+show.executions.view=true
```
Comment (FareesHussain) [inline .settings/org.eclipse.buildship.core.prefs]: revert these changes

**Review by FareesHussain — CHANGES_REQUESTED** [review]
Avoid additional changes that are auto-generated by the IDE as per the 4th point in the checklist
```
The PR does not contain any unnecessary auto-generated code from Android Studio.
```

File: .settings/org.eclipse.buildship.core.prefs (hunk 3 above)
Comment (Naman-1234) [reply .settings/org.eclipse.buildship.core.prefs]: Ok, sir, I didn't notice that. Thanks for telling me.

File: app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt [hunk 4]
```diff
@@ -118,12 +118,20 @@ class ProfileListFragmentTest {
         )
       )
       onView(
-        atPositionOnView(R.id.profile_list_recycler_view, 0, R.id.profile_list_name)
+        atPositionOnView(
+          recylerViewId = R.id.profile_list_recycler_view,
```
Comment (FareesHussain) [inline app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt]: A typo here, similarly for all the below arguments 

```suggestion
          recyclerViewId = R.id.profile_list_recycler_view,
```

Comment (Naman-1234) [reply app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt:122]: Ok sir , Will do that, And Could you tekk mw how could I revert commit changes in my Pr, I know it locally but don't know about how to do it in a PR.

Comment (Naman-1234) [reply app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileListFragmentTest.kt]: @FareesHussain  Are changes fine now Sir?

**Comment by Naman-1234** [comment]
@anandwana001, Sir I think this issue is solved, Please review this PR and let me know if there is any changes required.

**Comment by FareesHussain** [comment]
@Naman-1234 Please create a new PR as I will have to close this PR because of force-push. Force push tampers the commit history which is not correct and it becomes really hard to review such PRs. Also have a look at the wiki instructions its mentioned in there (5. point) that force push is not allowed. https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

Thanks.
